src/main/java/com/mabl/integration/jenkins/domain/ExecutionResult.java

```diff
@@ -132,4 +137,35 @@ public PlanExecutionResult(
             this.id = id;
         }
     }
+
+    @SuppressWarnings("WeakerAccess")
+    public static final class EventStatus {
+        public Boolean succeeded;
+        public Boolean succeededFirstAttempt;
+
+        public EventStatus(
+        ) {
+        }
+
+        @JsonGetter("succeeded")
+        public Boolean getSucceeded() {
+            return succeeded;
+        }
+
+        @JsonGetter("succeeded_first_attempt")
+        public Boolean getSucceededFirstAttempt() {
+            return succeededFirstAttempt;
+        }
+
+        @JsonSetter("succeeded")
+        public void setSucceeded(Boolean succeeded) {
```

Comment:
Do we need these setters? Seems to be a different pattern than other objects above.

Reply:
Yes, do we need them. The execution result API does not return event status until all plans finished. This allows delayed setting of these properties (as opposed to creating them at construction time). This is also the reason why the allPlansSuccess method in MablStepDeploymentRunner had to be updated to use a Boolean object.